src/mesh/mesh_pack.hpp

```diff
@@ -101,13 +98,13 @@ auto PackMesh(const Mesh *pmesh, F &packing_function) {
   Kokkos::deep_copy(packs, packs_host);
   Kokkos::deep_copy(coords, coords_host);
 
-  return MeshPack<T>(packs, pmesh->pblock->cellbounds, coords, dims);
+  return MeshPack<T>(packs, pmesh->block_list.front().cellbounds, coords, dims);
```

Comment:
Not your fault, but I really had this `block_list.front().blah` stuff. It's like, hm... should this attribute really hang off the `MeshBlock`?

Reply:
Yeah I was thinking about that actually. Was considering fixing that in another PR.